### ISSUE TITLE
Does neural style give comparable results to fast neural style tensorfire?

### ISSUE THREAD

**Issue by lavanyakapahi** [anishathalye/neural-style#125]
Hi Anish,
I am interested in experimenting with artistic style transfer and found your project. Thanks! I used the great wave attached as the style and ran it through 1000 iterations. The results are attached.
I then checked with TensorFire. The results are very nice! Can I achieve the same via neural style? I tried tuning the parameters but could not get good results.
Content image
![content_boy](https://user-images.githubusercontent.com/42534281/44337003-d4615b80-a496-11e8-8043-8d8aae510f75.JPG)
Style image
![style-wave](https://user-images.githubusercontent.com/42534281/44337007-d88d7900-a496-11e8-85fd-1c44afeabe2b.jpg)
Result after 1000 iterations
![1-boy-output](https://user-images.githubusercontent.com/42534281/44337182-55205780-a497-11e8-864b-04dde52b9532.jpg)
Result with TensorFire
<img width="465" alt="boy_wave_tensorfire" src="https://user-images.githubusercontent.com/42534281/44337059-f9ee6500-a496-11e8-861a-965fa315dd53.png">

Thanks so much
Lavanya

**Comment by anishathalye** [comment]
I don't know, I haven't done a very careful comparison. I think it's generally easier to get nicer results with [fast style transfer](https://github.com/lengstrom/fast-style-transfer) (which is what TensorFire uses).

**Comment by lavanyakapahi** [comment]
Thanks for suggesting the fast style transfer. I checked it and the pre trained model gave a very nice result. I need to train a new water colour style. I thought it would be a good idea to understand the tuning parameters by using the neural style before trying to train a model for the fast neural style. Not sure if that would be a right approach. So meanwhile, I continue trying out the neural style. I noticed that each time I run the script for 1000 iterations I get inconsistent results even though the parameters are the same. I ran the script with --content-weight 2 and --style-weight 1, twice. I have attached the results below. Is that expected?
![3-testoutput](https://user-images.githubusercontent.com/42534281/44484014-34f4c200-a66a-11e8-89aa-124c4a337037.jpg)
![3-testoutput_2](https://user-images.githubusercontent.com/42534281/44484022-39b97600-a66a-11e8-9eb5-48a5989b0d37.jpg)
